harvester/actor.py

```diff
@@ -64,14 +105,23 @@ def to_metax_dict(self):
         testaibility. Performance hit should be minimal, as the lists are tiny (4 items
         at most).
         """
-        return {"roles": sorted(list(self.roles)), "person": self._person_dict}
+        return {
+            "roles": sorted(list(self.roles)),
```

Comment:
Btw `sorted()` returns a list, so `list()` is redundant.
```suggestion
            "roles": sorted(self.roles),
```